k8s/models/common.py

```diff
@@ -50,6 +50,24 @@ class ObjectMeta(Model):
     selfLink = ReadOnlyField(six.text_type)
     uid = ReadOnlyField(six.text_type)
 
+class ObjectReference(Model):
+    apiVersion = Field(six.text_type)
+    fieldPath = Field(six.text_type)
+    kind = Field(six.text_type)
+    name = Field(six.text_type)
+    namespace = Field(six.text_type)
+    resourceVersion = Field(six.text_type)
+    uid = Field(six.text_type)
+
+class ListMeta(Model):
+
+    _continue = Field(six.text_type) 
+    remainingItemCount = Field(int)
```

Comment:
Codacy found an issue: [variable 'remainingItemCount' in class scope should not be mixedCase (N815)](https://app.codacy.com/manual/fiaas/k8s/pullRequest?prid=5919885)